grakn-dist/src/main/java/ai/grakn/dist/DistGrakn.java

```diff
@@ -0,0 +1,719 @@
+            }
+            try {
+                Thread.sleep(WAIT_INTERVAL_S * 1000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
+
+        if(Files.exists(Paths.get(GRAKN_PID))) {
+            try {
+                Files.delete(Paths.get(GRAKN_PID));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        System.out.println("FAILED!");
+        System.out.println("Unable to start Grakn");
+    }
+
+    private boolean graknCheckIfReady(String host, int port, String path) {
+        try {
+            URL siteURL = UriBuilder.fromUri(new SimpleURI(host, port).toURI()).path(path).build().toURL();
+            HttpURLConnection connection = (HttpURLConnection) siteURL
+                    .openConnection();
+            connection.setRequestMethod("GET");
+            connection.connect();
+
+            int code = connection.getResponseCode();
+            if (code == 200) {
+                return true;
+            } else {
+                return false;
+            }
+        } catch (IOException e) {
+            return false;
+        }
+    }
+
+    private void startAll() {
+        startStorage();
+        startQueue();
+        startGrakn();
+
+        if(!graknIsStarted || !queueIsStarted || !storageIsStarted) {
+            System.out.println("Please run 'grakn server status' or check the logs located under 'logs' directory.");
+        }
+
+    }
+
+    public void graknStopProcess() {
+        String graknPid="";
+        if(Files.exists(Paths.get(GRAKN_PID))) {
+            try {
+                graknPid = new String(Files.readAllBytes(Paths.get(GRAKN_PID)),StandardCharsets.UTF_8);
+                graknPid = graknPid.trim();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+            if (graknPid.trim().isEmpty()) {
+                return;
+            }
+            processHandler.kill(graknPid);
+        }
+
+        while(true) {
+            System.out.print(".");
+            System.out.flush();
+
+            OutputCommand outputCommand = processHandler.executeAndWait(new String[]{
+                    "/bin/sh",
+                    "-c",
+                    "kill -0 " + graknPid.trim()
+            }, null, null);
+
+            if(outputCommand.exitStatus==0) {
+                System.out.println("SUCCESS");
+                try {
+                    Files.delete(Paths.get(GRAKN_PID));
+                } catch (IOException e) {
+                    e.printStackTrace();
+                }
+                return;
+            }
+            try {
+                Thread.sleep(WAIT_INTERVAL_S * 1000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private void queueStopProcess() {
+        String queuePid="";
+        if(Files.exists(Paths.get(QUEUE_PID))) {
```

Comment:
I think this would be clearer flipped:

```
if (!Files.exists(Paths.get(QUEUE_PID))) {
    return;
}

...

if (queuePid.isEmpty()) {
    return;
}
```